### PR TITLE
docs: remove duplication in shared client extensions pages

### PR DESCRIPTION
**Pull request by ruheni**
- Link out to `query` extension docs
- Remove redundant information

### CODE REVIEW

**Review by janpio — COMMENTED** [review]
See https://prisma-company.slack.com/archives/C043X6JBND6/p1693838821507319?thread_ts=1691704235.991279&cid=C043X6JBND6 and following

**Review by nikolasburk — COMMENTED** [review]
this looks good to me at first sight! anything we still need to fix here or is this one ready to merge?

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/140-shared-extensions.mdx [hunk 1]
```diff
@@ -13,20 +13,47 @@ If you would like to build a shareable extension, we also recommend using the [`
...
+```ts
+import findOrCreate from 'prisma-extension-find-or-create'
+
+const prisma = new PrismaClient().$extends(findOrCreate)
+const user = await prisma.user.findOrCreate()
+```
+
+When you call a method in an extension, use the constant name from your `$extends` statement, not `prisma`. In the above example,`xprisma.user.findOrCreate` works, but `prisma.user.findOrCreate` does not, because the original `prisma` is not modified.
+
+## Create a sharable extension
```
Comment (nikolasburk) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/140-shared-extensions.mdx]:
```suggestion
## Create a shareable extension
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/140-shared-extensions.mdx [hunk 2]
```diff
@@ -93,8 +97,14 @@ export default Prisma.defineExtension({
 
 </details>
 
+### Publishing the shareable extension to npm
+
 You can then share the extension on `npm`. When you choose a package name, we recommend that you use the `prisma-extension-<package-name>` convention, to make it easier to find and install.
 
+## Other considerations
+
+bla bla, other things to be aware or take advantage of
```
Comment (nikolasburk) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/140-shared-extensions.mdx]: this needs to be resolved it seems like, maybe just drop the entire section if we have no content for it?

Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/140-shared-extensions.mdx]:
```suggestion
```

**Review by nikolasburk — APPROVED** [review]

**Pull request merged**